### PR TITLE
Fix a few spelling mistakes

### PR DESCRIPTION
**Pull request by alexwlchan**

### CODE REVIEW

File: _posts/2016-12-05-integrated-shrinking.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ Haskell](https://mail.haskell.org/pipermail/libraries/2013-November/021674.html)
 and [Jack](https://github.com/ambiata/disorder.hs/tree/master/disorder-jack) is
 an alternative property based system for Haskell that does this - but you
 feel the pain much more quickly in dynamic languages because the typical
-workaround for this problem in Haskell is to define a newtype, which lets you
+workaround for this problem in Haskell is to define a new type, which lets you
```
Comment (DRMacIver) [inline _posts/2016-12-05-integrated-shrinking.md]: This is actually not a spelling mistake, but a Haskell specific term. newtype is the keyword used for defining lightweight wrapper types that do nothing but wrap a single value of another type.

File: _posts/2016-12-05-integrated-shrinking.md [hunk 2]
```diff
@@ -155,7 +155,7 @@ your shrinkers, which is more work and more brittle than just having it work
 automatically.
 
 So although (as far as I know) none of the currently popular property based
-testing langugages for statically typed languages implement this behaviour
+testing languages for statically typed languages implement this behaviour
```
Comment (DRMacIver) [inline _posts/2016-12-05-integrated-shrinking.md]: I think languages is just the wrong word altogether here actually. I guess I meant systems?

**Review by DRMacIver — CHANGES_REQUESTED** [review]
Thanks for the corrections, despite the feedback! Would you like to fix the fixes or shall I?

**Comment by alexwlchan** [comment]
Fixes made.

**Review by DRMacIver — APPROVED** [review]

**Comment by DRMacIver** [comment]
Thanks!

**Pull request merged**